### DISCUSSION TITLE
Level list and their spawns

### DISCUSSION THREAD

**Discussion by slysnivy** [shoc71/dont-stop-now#128] [General]
List of levels and their spawn
TutorialLevel1(12, 320, 1) - Hallway - needs more decorations
TutorialLevel2(12, 80, 1) - Dropguide to Solution - needs more decorations
TutorialLevel3(12, 297, 1) - Jump Over the Box - needs more decorations
TutorialLevel4(12, 200, 1) - Mind the Gap - move the text & more decorations
EasyLevel1(12, 292, 1) - Candles - maybe needs more decorations
EasyLevel2(132, 282, 1) - Mouse - I think it's good - would be nice if we had a sfx play if the player collides with the mouse nose
EasyLevel3(12, 12, 1) - Block Maze 5 - moved to another location or had as a failed level concept
EasyLevel4(37, 489, 1) - Letters - I think this is good - might more coloring with white blocks
EasyLevel5(30, 518, 1) - Staggers - - [ ] would be nice if I could add some blue and white and yellow boxes in the level to show so lights or something
EasyLevel6(21, 21, 1) - Spiral - made for testing purposes only. Needs more polish for it to be a playable level for the public.

--------NEEDS MORE TESTING__
EasyLevel7(37, 527, 1) - Burger - needs more dull coloring to avoid confusion
EasyLevel8(21, 258, 1) - Tip-toe - boxes need readjustments and better positioning
EasyLevel9(37, 524, 1) - You're not alone - boxes need readjustments and better positioning, more decorations for people and "platforms"
EasyLevel10(48, 53, 1) - Ritual or (keys) - boxes need readjustments and better positioning, more text, and renamed to Hardlevel1 sonn...
EasyLevel11(32, 338, 1) - Reverse Lamp - I think this one is solid and good
EasyLevel12(37, 47, 1) - Propaganda - This is a hard one and needs more detailed block as well towards the bottom
EasyLevel13(23, 259, 1) - Split Path - this one is eh
EasyLevel14(69, 263, 1) - Uneven platforms - could use more decorations and movements for improvements
EasyLevel15(6, 272, 1) - Lava drip - lvl just needs more testing and adjusting
EasyLevel16(33, 44, 1) - Ashtray - we good here

EasyLevel17(37, 47, 1) - ribbons - we good
EasyLevel18(37, 47, 1) - Choose Bait - better decorations at the bottom
EasyLevel19(26, 62, 1) - Upper Gates - more jumping and decorations needed
EasyLevel20(48, 53, 1) - na
EasyLevel21(32, 338, 1) - na
EasyLevel22(37, 47, 1) - na
EasyLevel23(23, 259, 1) - na
EasyLevel24(69, 263, 1) - na
EasyLevel25(6, 272, 1) - na
EasyLevel26(33, 44, 1) - na
EasyLevel27(37, 527, 1) - na
EasyLevel28(21, 258, 1) - na
EasyLevel29(37, 524, 1) - na
EasyLevel30(48, 53, 1) - na
EasyLevel31(32, 338, 1) - na
EasyLevel32(37, 47, 1) - na
EasyLevel33(23, 259, 1) - na
EasyLevel34(69, 263, 1) - na
EasyLevel35(6, 272, 1) - na
EasyLevel36(33, 44, 1) - na

____Unorgainzed levels
TutorialLevel14(33, 44, 1) - daz maze- no known spawn, figure it out
TutorialLevel24(0, 300, 1) - red floor- we good here
TutorialLevel5(10, 100, 1) - sandwich- no known spawn, figure it out
TutorialLevel6(0, 140, 1) - Not the same - _more blocks here needed _
TutorialLevel7(0, 120, 1) - Jump under the platform - we good here
HardLevel1(33, 44, 1) - Small Steps - figoure spawn out and difficulty of level

Note to self and others who might read this.
This issue shouldn't be resolved until all the levels have been tested and approved for functionality, difficulty, and decorations.
This list will be updated and modified when improvements are made with it.

**Comment by slysnivy** [comment]
If there are any issues with the level names, you can create an issue from this discussion

**Comment by shoc71** [comment]
I don't think there is going to be an issue with the level names and all that, and yes like you said we can always open up an issue. There is another issue very similar to this #205 that is opened up for this that not only summarizes everything here but does a better job encompassing and organizing all the faults and better ideas we have for this issue/discussion.
